hazelcast/src/main/java/com/hazelcast/nio/tcp/PacketDecoder.java

```diff
@@ -29,24 +29,23 @@
 /**
  * The {@link ChannelInboundHandler} for member to member communication.
  *
- * It reads as many packets from the src ByteBuffer as possible, and each of the Packets is send to the {@link PacketHandler}.
+ * It reads as many packets from the src ByteBuffer as possible, and each of the Packets is send to the handler.
```

Comment:
Should be `{@link Consumer}`, like in the other Javadocs. Also `is sent`, not `is send`.